### PR TITLE
Add a changelog

### PR DESCRIPTION
**Pull request by SemenchenkoVitaliy**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+### Added
+
+- The first implementation of the package.
+
+[unreleased]: https://github.com/metarhia/metadoc/compare/v0.5.1...HEAD
+[0.5.1]: https://github.com/metarhia/metadoc/compare/v0.5.0...v0.5.1
+[0.5.0]: https://github.com/metarhia/metadoc/compare/v0.4.1...v0.5.0
+[0.4.1]: https://github.com/metarhia/metadoc/compare/v0.4.0...v0.4.1
+[0.4.0]: https://github.com/metarhia/metadoc/compare/a7a7ca1...v0.4.0
+[0.3.0]: https://github.com/metarhia/metadoc/compare/v0.2.1...a7a7ca1
```
Comment (belochub) [inline CHANGELOG.md]: Should v0.3.0 be tagged maybe?

Comment (SemenchenkoVitaliy) [reply CHANGELOG.md]: There is no `v0.3.0` release. See https://github.com/metarhia/metadoc/releases.

Comment (belochub) [reply CHANGELOG.md]: Yes, that's why I've proposed to tag v0.3.0.

**Comment by lundibundi** [comment]
Landed in https://github.com/metarhia/metadoc/commit/3f1705ea4ad8f0ff2c4226f7b3d4fad60155f832.

**Pull request closed without merging**